drkafka/pom.xml

```diff
@@ -167,7 +167,7 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>${fasterxml.version}</version>
+			<version>2.9.9.1</version>
```

Comment:
do we also need to change this line?

Reply:
2.9.9.1 is available for `jackson-databind` only. The other packages don't have 2.9.9.1 so we hardcode it here for now